docs/test_cases/TestSuite42.md

```diff
@@ -0,0 +1,194 @@
+assignment for the same BIE and type code already exists!. (Assertion [#8](#test-assertion-4228)).
+14. The end user goes back to "Business Term Assignments" from the BIE dropdown list. 
+15.He clicks on the "Assign Business Term" button. Then select any BIE in the table and click "Next". 
+He will select any business term in the new table, put a type code and clicks on "Create" button. Then repeat the whole process for the same BIE with the same business term and using a different type code. Verify that the second time he clicks on the "Create" button, the assignment is created. (Assertion [#9](#test-assertion-4229)).
+
+
+## Test Case 42.3
+
+**Business Term from BIE detail page **
+
+Pre-condition: N/A
+
+### Test Assertion:
+
+#### Test Assertion #42.3.1
+From View/Edit BIE page, select the desired BIE, select the desired node under the BIE, the end user can click the button "Show Business Terms" to view all the business term assignments for that node. 
+
+#### Test Assertion #42.3.2
+
+From View/Edit BIE page, select the desired BIE, select the desired node under the BIE, the end user can hover over the button "Show Business Terms" without clicking, the business term assignments for that node will show up. 
+
+#### Test Assertion #42.3.3
+
+From View/Edit BIE page, select the desired BIE, select the desired node under the BIE, the end user can click the button "Assign New Business Term" to assign new business terms to that node. 
+
+
+### Test Step Pre-condition:
+1. There are at least three BIEs created by end user and developer and they are in different states (WIP, QA, Production). 
+2. There are at least five business terms created. Three of them are assigned to different BIEs. At least Two of them are not assigned to any BIEs. 
+
+
+### Test Step:
+1. An end user logins.
+2. He visits the Home page and clicks on the BIE tab.
+3. He clicks on the "View/Edit BIE" from the dropdown list.
+4. He clicks on any BIE from the table. The BIE detail page will open.
+5. He expands the root level in the left sidenav tree and clicks on any used node in the second level, the detail for that node will disply on the right.
+6. He will click on "Show Business Terms" button to view all the business terms that already assigned to that node.
+7. Verify that the list of business terms in the new window is indeed assigned to the selected node (Assertion [#1](#test-assertion-4231)).
+8. He closes the new window and goes back to previous window.
+9. He will hover over "Show Business Terms" button without clicking. A hint bar will display to show the number of business terms assigned to the node. Verify the number is the same as in the previosu step. (Assertion [#2](#test-assertion-4232)).
+10. He will clicks on the "Assign New Business Term" button right next to "Show Business Term" button on the same page.
```

Comment:
'Assign New Business Term' button won't be activated if the BIE 'Used' checkbox is not enabled. Because the BIE record is not existed in DB until it checked.

Reply:
I mention that the end user will select a used node.

Reply:
I fixed this issue.